docs/user/manual.adoc

```diff
@@ -234,16 +234,43 @@ $ brew install rust-analyzer
 
 === Emacs
 
-Note this excellent https://robert.kra.hn/posts/2021-02-07_rust-with-emacs/[guide] from https://github.com/rksm[@rksm].
-
 Prerequisites: You have installed the <<rust-analyzer-language-server-binary,`rust-analyzer` binary>>.
 
-Emacs support is maintained as part of the https://github.com/emacs-lsp/lsp-mode[Emacs-LSP] package in https://github.com/emacs-lsp/lsp-mode/blob/master/lsp-rust.el[lsp-rust.el].
+To use `rust-analyzer`, you need to install and enable one of the two popular two popular LSP client implementations for Emacs, https://github.com/joaotavora/eglot[Eglot] or https://github.com/emacs-lsp/lsp-mode[LSP Mode]. Both enable `rust-analyzer` by default in rust buffers if it is available.
+
+==== Eglot
+
+Eglot is the more minimalistic and lightweight LSP client for Emacs, integrates well with existing Emacs functionality and will be built into Emacs starting from release 29.
+
+After installing Eglot, e.g. via `M-x package-install` (not needed from Emacs29), you can enable it via the `M-x eglot` command or load it automatically in `rust-mode` via
```

Comment:
Nit (unless it's actually supposed to be spelled like that):

```suggestion
After installing Eglot, e.g. via `M-x package-install` (not needed from Emacs 29), you can enable it via the `M-x eglot` command or load it automatically in `rust-mode` via
```

Reply:
Just checked that the Eglot manual writes it with a space too. I commited the suggestion via the github UI and then locally squashed via an interactive rebase and force-pushed then again. Not sure if that is good practice, usually I don't force-push on branches which are in an active PR, but I didn't know how to squash otherwise, except in the github merge button options.